conf/application.conf

```diff
@@ -66,6 +66,11 @@ microservice {
       start.path = "/agent-subscription/start"
     }
 
+    agent-invitations-frontend {
+      external-url = "http://localhost:9448"
+      start.path = "/agent-invitations/agents"
```

Comment:
You have to change the path to "/invitations/agents/" 
See: https://github.com/hmrc/agent-invitations-frontend/blob/master/conf/prod.routes#L2